packages/terra-form-select/src/combobox/Frame.jsx

```diff
@@ -348,7 +353,7 @@ class Frame extends React.Component {
     // Don't blur if we dismissed the onscreen keyboard
     // Determined by if we have have interacted with the frame via onTouchStart
     // and if the focus is on input.
-    if (focusedByTouch && (relatedTarget === this.input)) {
+    if (relatedTarget === this.input) {
```

Comment:
don't we need `focusedByTouch` here to differentiate between touch and non-touch device...??

Reply:
This change is done so that the dropdown does not close when user opens it via toggle and clicks on input to focus it in desktop. No side effects in mobile.